packages/vm/src/dynamic_link.rs

```diff
@@ -409,7 +412,7 @@ mod tests {
             let mut caller_env = &mut caller_instance.borrow_mut().env;
             let target_func_info = FunctionMetadata {
                 module_name: CALLER_NAME_ADDR.to_string(),
-                name: "foo".to_string(),
+                name: "fail".to_string(),
```

Comment:
do not change (cause of failure should be not WASM but address)

Reply:
Fixed.